apps/web/src/components/easter-eggs/subway-surfers.tsx

```diff
@@ -0,0 +1,51 @@
+import { useEffect, useRef, useState } from "react";
+import Image from "next/image";
+
+import SubwaySurfersImage from "@/assets/easter-eggs/subway-surfers.webp";
+
+const SHORTCUT_KEYS = ["s", "u", "r", "f"];
+
+export default function SubwaySurfers() {
+  const [easterEggSettingsVisible, setEasterEggSettingsVisible] =
+    useState(false);
+
+  const pressed = useRef<{ [key: string]: boolean }>({});
+
+  useEffect(() => {
+    const handleKeyDown = (e: KeyboardEvent) => {
+      const key = e.key.toLowerCase();
+      if (SHORTCUT_KEYS.includes(key)) {
+        pressed.current[key] = true;
+        if (SHORTCUT_KEYS.every((k) => pressed.current[k])) {
+          setEasterEggSettingsVisible(true);
+        }
+      }
+    };
+
+    const handleKeyUp = (e: KeyboardEvent) => {
+      const key = e.key.toLowerCase();
+      if (SHORTCUT_KEYS.includes(key)) {
+        pressed.current[key] = false;
+        setEasterEggSettingsVisible(false);
+      }
+    };
+
+    window.addEventListener("keydown", handleKeyDown);
+    window.addEventListener("keyup", handleKeyUp);
+
+    return () => {
+      window.removeEventListener("keydown", handleKeyDown);
+      window.removeEventListener("keyup", handleKeyUp);
+    };
+  }, []);
```

Comment:
`useHotkeys("s+u+r+f", ...)`

Reply:
I actually had that but it doesnt work well sadly

Reply:
See discord for me explaining the issue